### DISCUSSION TITLE
Unable to select table in target view of Schema Compare tool

### DISCUSSION THREAD

**Discussion by jenomot** [dbeaver/dbeaver#17902] [Q&A]
System information: 
Operating system (distribution) and version: Win 10
DBeaver version: 22.2.0.202209090951
Additional extensions
Connection specification:
Database name and version: Teradata 16.20
Driver name: Teradata 17.20.00.12
Do you use tunnels or proxies (SSH, SOCKS, etc)? No
Describe the problem you're observing:
When I use the Schema Compare tool, I am able to select a table in the source section but in the target section, I am only able to select the entire database. I would like to be able to select the individual tables in both the source and the target section so I can compare the schemas in a particualr table. This occurs for any database that I bring up in the target section, whether it be Teradata or SQL. If I swap the sources, the tables appear in the source section. 
Is there a setting that I'm missing?

**Comment by LonwoLonwo** [comment]
Hello @jenomot 

The Schema Compare tool only works for schemas/catalogs/table containers comparison.

But what is the particular reason to compare tables? You can open tables DDL tabs with the structure of the table as an example.

**Reply by jenomot** [reply]
Hi @LonwoLonwo our company is migrating from Teradata to SQL so I need to be able to verify that the schema of the tables created in SQL are comparable to the schema of the tables in Teradata. I understand that there will not be an exact match but I was hoping for a tool that will help point out any differences.
Thanks!
